configs/mainnet.json

```diff
@@ -17,31 +17,34 @@
     "usdc.1.6.e70e14"
   ],
   "transfer_options": {
-    "Osmosis": 0,
-    "Noble": 1,
-    "Cosmos Hub": 2,
-    "Evmos": 3,
-    "Axelar": 4,
-    "Juno": 5,
-    "Stride": 6,
-    "Kujira": 7,
-    "Terra": 8,
-    "Comdex": 9,
-    "Stafihub": 10,
-    "Persistence Core": 11,
-    "Stargaze": 12,
-    "Canto": 13,
-    "Sommelier": 14,
-    "IRIS Hub": 15,
-    "Ethereum": 16,
-    "Arbitrum": 17,
-    "Polygon": 18,
-    "OKC": 19,
-    "Binance Smart Chain": 20,
-    "Carbon": 21,
-    "Neo": 22,
-    "Neo3": 23,
-    "Zilliqa": 24
+    "Mantle": 0,
+    "OP": 1,
+    "Base": 2,
```

Comment:
Think you can move `Mantle`, `OP` and `Base` below `IRIS Hub` together with the rest of the EVM chains

Reply:
Checked with CJ to bump new chains to top of transfers options